This patch adapts the tbb_cmakeConfig.patch for 2019.5 and later.
The file changed.

--- cmake/TBBMakeConfig.cmake.orig	2019-03-21 10:28:21.000000000 -0500
+++ cmake/TBBMakeConfig.cmake	2019-11-27 22:52:45.270824744 -0600
@@ -175,7 +175,7 @@
 
 ${TBB_CHOOSE_COMPILER_SUBDIR}
 
-get_filename_component(_tbb_lib_path \"\${_tbb_root}/${TBB_SHARED_LIB_DIR}/\${_tbb_arch_subdir}/\${_tbb_compiler_subdir}\" ABSOLUTE)
+get_filename_component(_tbb_lib_path \"\${_tbb_root}/${TBB_SHARED_LIB_DIR}\" ABSOLUTE)
 ")
 
     set(TBB_RELEASE_LIB_PATH "\${_tbb_lib_path}")
